.github/workflows/cicd.yaml

```diff
@@ -210,7 +210,7 @@ jobs:
     container:
       image: ghcr.io/4paradigm/hybridsql:0.4.0
     if: >
-      success() && startsWith(github.ref, 'refs/tags/v')
+      success() && github.event_name == 'push'
```

Comment:
It's better to add a comment here, to explain why java sdk trigger event is different